Plugin load error [zeit/hyper/issues/191]

Hello,

I don't know why, I get this error, and the theme don't work :/


```
Error: Cannot find module '/Users/remivillien/.hyper_plugins/node_modules/hyper-material-theme'
    at Module._resolveFilename (module.js:455:15)
    at Function.Module._resolveFilename (/Applications/Hyper.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:403:25)
    at require (internal/module.js:20:19)
    at le.plugins.concat.map.ce (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:8876)
    at Array.map (native)
    at se (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:8830)
    at Object.<anonymous> (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:10365)
    at __webpack_require__ (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:119)
    at Object.<anonymous> (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:47:368728)le.plugins.concat.map.ce @ bundle.js:1
bundle.js:1 [Notification] Plugin load error: "hyper-material-theme" failed to load in the renderer process. Check Developer Tools for details.

[Notification] Plugin load error: "hyper-material-theme" failed to load in the renderer process. Check Developer Tools for details.

index.html:35 total init time 162.32500000000002

bundle.js:47 Unimplemented DEC Private Mode: 1034```

--- a/index.js
+++ b/index.js
@@ -6,8 +6,8 @@
 
 exports.decorateConfig = config => {
 
-  const ThemeConfig = config.MaterialTheme;
-  var ThemeBackground;
+  const ThemeConfig = config.MaterialTheme || {};
+  let ThemeBackground;
 
   // Set vibrancy
   exports.onWindow = (browserWindow) => {
@@ -24,7 +24,7 @@ exports.decorateConfig = config => {
     ThemeBackground = `rgba(38, 50, 56, ${ThemeConfig.backgroundOpacity || '1'})`;
   }
 
-  config.backgroundColor = ThemeBackground;
+  config.backgroundColor = ThemeBackground || config.backgroundColor;
   config.foregroundColor = '#ECEFF1';
   config.borderColor = '#37474F';
   config.cursorColor = `${config.cursorColor || '#FFCC00'}`;

